### ISSUE TITLE
[blocking] unable to deploy VA due to botservice compatibility errors

### ISSUE THREAD

**Issue by stepknees** [Microsoft/botbuilder-tools#886]
**Project**
Virtual Assistant

**Description**
While working through initial VA deployment steps, I'm not able to add the botservice via Azure CLI. This then causes the deploy_bot.ps1 script to fail. 

**To Reproduce**
az extension add -n botservice
>Skipping 'botservice-0.4.3-py2.py3-none-any.whl' as not compatible with this version of the CLI. Extension compatibility result: is_compatible=False cli_core_version=2.0.54 min_required=2.0.52 max_required=2.0.52

**Expected behavior**

**Screenshots**

**Additional context**
Azure CLI version is azure-cli (2.0.54)

msbot -v 
4.3.1

**Comment by stevengum** [comment]
Closing as this has been addressed with the latest msbot version [**4.3.2**](https://www.npmjs.com/package/msbot/v/4.3.2) as indicated in the above comment.

If you still run into this issue, please **@mention** me in this issue or open up a new issue.

**Comment by stepknees** [comment]
Thanks – trying this out right now and it seems to have unblocked me.
